subsys/net/ip/net_stats.h

```diff
@@ -308,6 +310,28 @@ static inline void net_stats_update_ipv6_mld_drop(struct net_if *iface)
 #define net_stats_update_ipv6_mld_drop(iface)
 #endif /* CONFIG_NET_STATISTICS_MLD */
 
+#if defined(CONFIG_NET_CONTEXT_TIMESTAMP) && defined(CONFIG_NET_STATISTICS)
+static inline void net_stats_update_tx_time(struct net_if *iface,
+					    u32_t start_time,
+					    u32_t end_time)
+{
+	u32_t diff = (u32_t)abs((s32_t)end_time - (s32_t)start_time);
```

Comment:
Some MISRA pundit might say you're calling for undefined behavior here. Instead, you should subtract unsigned, and convert the result to s32_t.